microsoft-edge/webview2/get-started/hololens2.md

```diff
@@ -458,10 +401,28 @@ When developing a HoloLens 2 Unity app with WebView2, be aware of some limitatio
 * **Performance**: Complex websites with heavy use of JavaScript or advanced rendering may impact system performance or the host application's framerate. For general performance-related limitations and recommendations, see [Understanding performance for mixed reality](/windows/mixed-reality/develop/advanced-concepts/understanding-performance-for-mixed-reality) in the Mixed Reality documentation. Also see [Performance optimization](#performance-optimization), below.
 
 
+<!-- ------------------------------ -->
+#### Performance optimization
+
+Optimizing the performance of WebView2 in your HoloLens 2 Unity app is crucial for a smooth user experience. Here are some recommendations:
+
+* **Limit the number of WebView2 instances**: We suggest using only one instance of WebView2 within a Unity app. Reuse the same instance or tear down and create a new one as needed. Keep in mind that removing the WebView prefab from the scene might not destroy the underlying WebView2 instance. You must call the `Destroy()` method on the game object to destroy it properly.
+
+* **Apply general Unity optimization techniques**: Utilize standard Unity optimization methods, such as occlusion culling or limiting the update rate, to improve WebView2 performance. For more information, see [Performance recommendations for Unity](/windows/mixed-reality/develop/unity/performance-recommendations-for-unity?tabs=openxr) in the Mixed Reality documentation.
+
+* **Profile and monitor WebView2 performance**: There are several ways to profile the performance of a HoloLens 2 Unity application:
+
+  * **Unity Profiler**: A built-in tool in Unity that allows you to measure and optimize the performance of your application on various platforms, including HoloLens 2.
+
+  * **Visual Profiler**: A feature of the Mixed Reality Toolkit (MRTK) that provides an in-application view of your application's performance.
+
+  * **PIX**: A performance tuning and debugging tool for Windows that can also be used to profile Unity applications on HoloLens 2.
+
+
 <!-- ------------------------------ -->
 #### Navigation
 
-In Step 6, we touched on some navigation methods. In this section, we'll expand on what we learned.
+In Step 7, we touched on some navigation methods. In this section, we'll expand on what we learned.
```

Comment:
```suggestion
In the [Step 7 - Fooing the bar](#foo) step, we touched on some navigation methods. In this section, we'll expand on what we learned.
```
I'd try to avoid depending only on number of a major Step.